garage/contrib/exp/checkpointers/checkpointer.py

```diff
@@ -0,0 +1,45 @@
+import abc
+
+
+def get_now_timestamp():
+    import datetime
+    return datetime.datetime.now().isoformat()
+
+
+def get_timestamp(name):
+    import re
+    isoformat_regex = r'\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+'
+    return re.search(isoformat_regex, name).group(0)
+
+
+def cat_for_fname(*args):
+    return '_'.join(args) + '.pkl'
+
+
+class Checkpointer(abc.ABC):
+    def __init__(self, prefix, resume=True):
+        self.resume = resume
+        self.prefix = prefix
+
+    @abc.abstractmethod
+    def load(self, **kwargs):
```

Comment:
is `**kwargs` necessary in an ABC if you want child classes to be able to add additional args?